configure.ml

```diff
@@ -1091,6 +1093,8 @@ let _ =
 
 (** * Build the config/Makefile file *)
 
+let opt s = if s = "" then s else " "^s
+
```

Comment:
Yes, I initially planned to avoid the two possible useless extra spaces in the CAMLFLAGS option (and using the function `opt` for this), then I saw @SkySkimmer's patch in trunk and decided to rather rely on his patch.

Now removed.